File: CourierApp.Shared.Old/DTOs/WorkStartDto.cs
using System.ComponentModel.DataAnnotations;

namespace CourierApp.Shared.DTOs;

public class WorkStartDto
{
    [Required]
    public int UserId { get; set; }
    [Required]
    public int PackageCount { get; set; }
    [Required]
    public double PricePerPackage { get; set; }
}

File: courier-app.server/src/work/dto/work-start.dto.ts
import { IsInt, IsNumber } from "class-validator";
import { ApiProperty } from '@nestjs/swagger';

export class WorkStartDto {
    @ApiProperty()
    @IsInt()
    userId: number;

    @ApiProperty()
    @IsInt()
    packageCount: number;

    @ApiProperty()
    @IsNumber()
    pricePerPackage: number;
}